src/redux/selectors/blockchainSelectors.js

```diff
@@ -12,12 +12,12 @@ const allWalletsSelector = createSelector(
 );
 const userWalletAddressSelector = createSelector(
   blockchainReducer,
-  (reducer) => reducer.userWalletAddress,
+  (reducer) => reducer?.userWalletAddress,
 );
 const isUserWalletAddressSameAsUserAdress = createSelector(
   (state) => state,
-  (reducer) => reducer.user.user.blockchainAddress
-      === reducer.blockchain.userWalletAddress,
+  (reducer) => reducer?.user?.user?.blockchainAddress
+      === reducer?.blockchain?.userWalletAddress,
```

Comment:
Ditto, `reducer`, `reducer.blockchain` and `reducer.user` should never be empty here. `reducer.user.user` is the only one that can be null in practice.
If that's not the case, there's a really nasty bug somewhere.